### DISCUSSION TITLE
Appointment Calander with different date and time

### DISCUSSION THREAD

**Discussion by anoop-mathew** [Budibase/budibase#6863] [Ideas]
Hi Team,

There is plugin call Amelia (https://wpamelia.com/), it's mainly for appointments in that a user can book appointments for different employees with there available time and date 
![Screenshot_20220722-195356](https://user-images.githubusercontent.com/55691873/180460546-9e9fe083-08b4-4f45-ad82-1575a7f03834.jpg)
Is there any option to have similar appointment functionality in budibase, i saw normal Calander is their based on each users available if their any open in future we can have a similar one

**Comment by melohagan** [comment]
Hey @anoop-mathew 

Custom components are on the roadmap, hopefully for release in September: https://github.com/Budibase/budibase/projects/10

When it's available it should be possible to include a library similar to this.
